.github/workflows/CIserver.yml

```
@@ -0,0 +1,74 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CIserver
```

ここでやりとりがあったので、名前を`CI-Server`にしておいてもらえると。
https://github.com/cocoa-mhlw/cocoa/pull/211#issuecomment-859553572

```suggestion
name: CI-Server
```

---

対応しました。

補足です。Serverの大文字小文字の間違い(コミットメッセージ側は正しい)に気づかずpushしたので、コミット履歴が2つになっています。分かりにくくなってしまいすみません…